### ISSUE TITLE
Tabs in high sierra

### ISSUE THREAD

**Issue by jhirn** [railwaycat/homebrew-emacsmacport#123]
Hi. Since installing High Sierra, I've been getting these tabs. Is there a way to disable them?

![screen shot 2017-12-07 at 10 48 46 am](https://user-images.githubusercontent.com/167089/33727004-574cd58a-db3c-11e7-86b6-75c9179f4816.png)

**Comment by P233** [comment]
You can disable the tab bar from the menu bar. 

![screen shot 2017-12-08 at 10 16 24 am](https://user-images.githubusercontent.com/1818679/33747977-ed26e8c4-dc00-11e7-8dc3-09e07d3aec50.png)

But I am wondering if there is a way to add a config in `.emacs.d`

**Comment by jhirn** [comment]
My hero. Thanks!!!!

**Comment by jhirn** [comment]
Assuming there’s a way to figure out how the menus are built to figure out the setq we deserve :)

**Comment by dotemacs** [comment]
Thanks for raising this issue @jhirn 👍 

I just experienced this too and then I remembered this issue.

I managed to turn off the tab with:

<kbd>M-x mac-toggle-tab-bar</kbd> 

Duckduckgo-ed around and found this StackOverflow thread that talks about the same issue:

https://stackoverflow.com/questions/46265615/emacs-mac-real-state/46274380

Looked at the source and this commit: 
https://bitbucket.org/mituharu/emacs-mac/commits/0817a1c5d018774c8dfe6deb6808fe78ac0d2b13
modifies `lisp/term/mac-win.el` which contains the above function.

What puzzled me is that I didn't have this option on, but then it just presented itself.
(After I `brew upgrade railwaycat/emacsmacport/emacs-mac` last night!)

Currently I'm on:
```emacs-25.3-mac-6.8```
<kbd>M-x emacs-version RET</kbd>
```GNU Emacs 25.3.1 (x86_64-apple-darwin15.6.0, Carbon Version 157 AppKit 1404.47) of 2017-11-14```
